guava/src/com/google/common/hash/Hashing.java

```diff
@@ -61,7 +65,9 @@
    * @return a hash function, described above, that produces hash codes of length {@code
    *     minimumBits} or greater
    */
-  public static HashFunction goodFastHash(int minimumBits) {
+  @SuppressWarnings("upperbound:array.access.unsafe.high.constant")// if lowest minimumBits is 1, checkPositiveAndMakeMultipleOf32(minimumBits) return
```

Comment:
Could you annotate `checkPositiveAndMakeMultipleOf32` to return `@IntRange(from=32)`?